examples/video/index.js

```diff
@@ -23,7 +23,8 @@ function onLoad() {
     height: 480,
     video: 'congo_2048.mp4',
     is_stereo: true,
-    loop: false,
+    loop: true,
+    //muted: false,
```

Comment:
if you uncomment this and set muted: true, then the video does start muted but the UI doesn't show the muted state. Can this be fixed easily?